doc/api/errors.md

```diff
@@ -253,6 +256,45 @@ will affect any stack trace captured _after_ the value has been changed.
 If set to a non-number value, or set to a negative number, stack traces will
 not capture any frames.
 
+### `error.cause`
+
+<!-- YAML
+added: v16.9.0
+-->
+
+* {any}
+
+The `error.cause` property is the cause of the error, as set by calling
```

Comment:
I suggest we re-use the wording from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause

```suggestion
The `cause` property indicates the specific original cause of an error.
It is used when catching and re-throwing an error with a more-specific or useful
error message in order to still have access to the the original error.
```

Reply:
I updated the PR with wording that is consistent with the other properties in this page while explaining the intent and use of the property like the MDN docs but in a Node-centric way that mentions error codes.